docs/src/pages/components/breadcrumbs/breadcrumbs.md

```diff
@@ -7,7 +7,7 @@ waiAria: https://www.w3.org/TR/wai-aria-practices/#breadcrumb
 
 # Breadcrumbs
 
-<p class="description">Breadcrumbs allow users to make selections from a range of values.</p>
+<p class="description">Breadcrumbs display and enable navigation up the application's hierarchy.</p>
```

Comment:
```suggestion
<p class="description">Breadcrumb navigation provides links back to each previous page the user navigated through, and shows the user's current location.</p>
```

Slightly adapted from [the W3 Breadcrumb description](https://www.w3schools.com/howto/howto_css_breadcrumbs.asp).

Reply:
@jamesmelzer Thanks for improving what is a very tenuous description - it barely seems to apply.

Doing a quick web search returns these descriptions:

- A website breadcrumb strategy is a visual navigation tool that helps users locate where they are within a website's content and how to return to an earlier page.

- Breadcrumbs in web design provide a visual representation of how a website is structured.

- Breadcrumbs are a structured series of links divided by a symbol. Each link first to last refers to the level of that page on your website.

- Breadcrumb navigation is a way for users to visualize their location on a website.

- Breadcrumbs are secondary elements of website navigation.

- A breadcrumb is a text path which reflects the site structure of the website. It tells users where they currently are on the site.

- Breadcrumbs are a navigational element that makes users aware of their current location within the hierarchical structure of a website. Breadcrumbs are a list of links representing the current page and its “ancestors” (parent page, grandparent page, and so on), typically going all the way back to the site homepage.

Picking out the keywords (navigation, visual/visualize/aware, locate/location/where, structure/level/hierarchy/path), and building on that, with particular reference to the last one (slightly paraphrased from Nielsen Norman):

```suggestion
<p class="description">Breadcrumbs consist of a list of links that help a user visualize a page's location within the hierarchical structure of a website, and allow navigation to any of its “ancestors” (parent page, grandparent page, and so on).
</p>
```

Reply:
@danilo-leal W3 School's description doesn't describe how most breadcrumbs work (rather how many users think they work).

Reply:
@mbrookes Thanks for the search! Your suggestion is good, but let's drop the parenthetical.

Reply:
```suggestion
<p class="description">Breadcrumbs consist of a list of links that help a user visualize a page's location within the hierarchical structure of a website, and allow navigation up to any of its “ancestors”.</p>
```